boxes/CMakeLists.txt

```diff
@@ -26,5 +26,5 @@ ADD_CUSTOM_TARGET(boxes
     )
 
 INSTALL(FILES download_boxes.sh make_boxes.sh utils.sh
-    DESTINATION ${INSTALLATION_DIR}/boxes
+    DESTINATION "${CMAKE_INSTALL_FULL_DATADIR}/sio2jail/boxes"
```

Comment:
Tak przy okazji, czy `make_boxes.sh` działa dobrze, gdy utils.sh nie jest w CWD?
(nie jest to uwaga do tej zmiany)

Reply:
Ależ owszem, jeszcze jak:
```

source "`dirname "${BASH_SOURCE}"`/utils.sh"

```